systemd/manageiq-providers-vmware_infra_manager_event_catcher@.service

```diff
@@ -3,10 +3,9 @@ PartOf=manageiq-providers-vmware_infra_manager_event_catcher.target
 [Install]
 WantedBy=manageiq-providers-vmware_infra_manager_event_catcher.target
 [Service]
-WorkingDirectory=/var/www/miq/vmdb
-Environment=BUNDLER_GROUPS=manageiq_default,ui_dependencies
-EnvironmentFile=/etc/default/manageiq*.properties
-ExecStart=/usr/bin/ruby lib/workers/bin/run_single_worker.rb ManageIQ::Providers::Vmware::InfraManager::EventCatcher --heartbeat --guid=%i
+WorkingDirectory=/opt/manageiq/manageiq-providers-vmware/workers/event_catcher
```

Comment:
So we're going to need a consistent directory here since we won't know the sha ahead of time

Reply:
Probably want a pattern in rpm build for putting non rails workers somewhere

Reply:
This directory might need productization (or maybe not).  Alternatively can we symlink to a known location and include the symlink in the RPM?

Reply:
Alternatively, can we change the directory of the process in the provider_worker and use something like __FILE__ in Ruby?  I'm thinking no, because we'd still have to know which event catcher to run.